boards/remote-revb/include/arduino_pinmap.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2020 J. David Ibáñez <jdavid.ibp@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     boards_remote-revb
+ * @{
+ *
+ * @file
+ * @brief       Mapping from MCU pins to Arduino pins for Remote revision B board
+ *
+ * You can use the defines in this file for simplified interaction with the
+ * Arduino specific pin numbers.
+ *
+ * @author      J. David Ibáñez <jdavid.ibp@gmail.com>
+ */
+
+#ifndef ARDUINO_PINMAP_H
+#define ARDUINO_PINMAP_H
+
+#include "periph/gpio.h"
+#include "periph/adc.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Mapping of MCU pins to Arduino pins
+ *
+ * @{
+ */
+
+#define ARDUINO_PIN_0           GPIO_PIN(PORT_A, 0) // UART0.RX
```

Comment:
Regarding this Arduino Pins: As far as I can tell, the board has no compatible layout for Arduino Shields. All other boards providing the Arduino feature are to some degree pin-compatible with the Arduino Uno / Arudino Mega2560 / Arduino Nano boards.

Is there a value in providing a pin mapping if there is no mechanical compatibility with Arduino Shields? If yes, I'm fine with this. But it would be nice to document this both here with a small comment here and maybe add something along the lines of the following to the board doc:

> Beware: Despite an Arudino pin mapping being available, Arduino shields are mechanically not compatible with the board. Check header file <LINK_TO_SOURCE_ON_GITHUB> for the exact mapping.

Reply:
Removing `arduino_pinmap` or leaving it empty makes the build to fail, at least pin0 has to be defined. Also defining the led makes `examples/arduino_hello-world` to work as expected, otherwise the led is not switched on/off at regular intervals. So I think there's still value to define the pinmap, at least partially, to get the examples/tests working.

The way I've defined the pins is 0/1 for RX/TX like in Arduino, then continue in order as the pins appear in https://github.com/Zolertia/Resources/wiki/RE-Mote#re-mote-revision-b-pin-out with the LEDs at the end. I can try to make the pins closer to those in Arduino as defined in https://www.arduino.cc/en/Reference/Board (e.g. SPI pins 10-13)